Python/Product/Debugger/Debugger/DebugEngine/AD7BoundBreakpoint.cs

```diff
@@ -51,7 +50,7 @@ class AD7BoundBreakpoint : IDebugBoundBreakpoint2 {
 
             if (!_deleted) {
                 _deleted = true;
-                _breakpoint.Remove();
+                TaskExtensions.RunSynchronouslyOnUIThread(ct => _breakpoint.RemoveAsync(ct));
```

Comment:
This isn't really an extension method anymore if we need to use the class name each time... do we have a Utilities or Helpers class? `*Extensions` should be almost entirely extension methods.